### DISCUSSION TITLE
My thank you discussion

### DISCUSSION THREAD

**Discussion by sachajw** [piraeusdatastore/piraeus-operator#753] [General]
Dear Piraeus Community,

Thank you so much for this fantastic project and for the effort put into it. I have been looking for something that can work on a combo of Pi4s and Pi5s on K3s with 3 USB 2TB disks and finally I found it here with Piraeus. I found the install straight forward and the documentation is great to get you going. I will be adding this to my blog post series on Ortelius the open source project I serve on which will be part of software defined storage. 

Thank you legends!

**Comment by WanzenBug** [comment]
Thank you for your kind words!

Was there any special configuration you had to do for K3s? If so, would you consider contributing it as a form of docs, like we have for example for [k0s](https://piraeus.io/docs/stable/how-to/k0s/)
